base_old_migration_fields/README.rst

```diff
@@ -0,0 +1,89 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+=========================
+Base Old Migration Fields
```

Comment:
You should create a README folder to autogenerate the readme file with the pre-commit command. 

Be careful with the format of the README folder files, is usually is .md instead of .rst and some syntax changes

See https://github.com/sygel-technology/sy-partner-contact/pull/46 as an example